.github/workflows/release.yml

```
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-18.04
     strategy:
       matrix:
-        go: ['1.14', '1.15', '1.16', '1.17']
+        go: ['1.14', '1.15', '1.16', '1.17', '1.18']
```

I would only support the last 3 versions (1.5 years)
```suggestion
        go: ['1.16', '1.17', '1.18']
```